package.json

```diff
@@ -17,12 +17,17 @@
     "cross-env": "^5.0.1",
     "express": "^4.15.3",
     "ipfs-api": "^22.0.1",
-    "nuxt": "^1.0.0-rc3",
-    "pg": "^7.4.3",
+    "node-sass": "^4.9.0",
+    "nuxt": "^1.4.0",
     "pg-hstore": "^2.3.2",
+    "pg": "^7.4.3",
+    "sass-loader": "^7.0.2",
     "sequelize": "^4.37.10",
     "sequelize-cli": "^4.0.0",
-    "source-map-support": "^0.4.15"
+    "source-map-support": "^0.4.15",
+    "vue": "^2.5.16",
```

Comment:
probably want to list `vue-server-renderer` and `vue-template-compiler` too if listing `vue` as dep,
since all 3 need to be sync in version


Another way is just use `// eslint-disable-line import/no-extraneous-dependencies` when ```import Vue from 'vue';``` in plugins,
since nuxt must have Vue anyway